Binary files orig/casper/src/.cstat.cpp.swp and patched/casper/src/.cstat.cpp.swp differ
diff -Nru orig/casper/src/cstat.cpp patched/casper/src/cstat.cpp
--- orig/casper/src/cstat.cpp	2023-10-20 20:19:10.000000000 +0200
+++ patched/casper/src/cstat.cpp	2024-03-16 11:20:09.456955900 +0100
@@ -9,9 +9,11 @@
 #include <stdlib.h>
 #include <string.h>
 #include <assert.h>
+
+#include "cstat.h"
+
 #include <R.h>
 #include <Rinternals.h>
-#include "cstat.h"
 
 static const char interface_c_sccs_id[] = "%W%";
 static const char mess_c_sccs_id[] = "%W%";
